doc/nrf/ug_thread_ot_integration.rst

```diff
@@ -0,0 +1,130 @@
+.. _openthread_integration:
+
+OpenThread Integration
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The goal of this page is to explain how the OpenThread stack is integrated with remaining parts of the system in terms of the file structure,
+used threads and synchronization primitives and the network traffic flow.
+
+The OpenThread stack is integrated with the Zephyr RTOS as the L2 layer with the IEEE 802.15.4 Radio Driver underneath and Zephyr IPv6 stack above it.
```

Comment:
Does it help with an L2/L3 + Zephyr IPv6/UDP/TCP diagram or is this obvious?

Reply:
Well, If I was looking at this page for the first time I would find this kind of description useful before starting the analysis of diagrams. Would you like to get rid of this part?

Reply:
No - I was suggesting adding a diagram of L2, L3 (I assume there is an L1) if that makes sense.

Reply:
For the end user/application developer is it not needed IMO. If the Radio Driver Team decides to add more documentation they could potentially write a few words about L1. We mostly work with L2/L3/L4. The first layer is a bit outside of my scope.